qldbdriver/result.go

```diff
@@ -108,3 +154,39 @@ func (result *BufferedResult) Next() bool {
 func (result *BufferedResult) GetCurrentData() []byte {
 	return result.ionBinary
 }
+
+// GetConsumedIOs returns the statement statistics for the total number of read IO requests.
+func (result *BufferedResult) GetConsumedIOs() *IOUsage {
+	return result.consumedIOs
+}
+
+// GetTimingInformation returns the statement statistics for the total server-side processing time.
+func (result *BufferedResult) GetTimingInformation() *TimingInformation {
+	return result.timingInformation
+}
+
+// IOUsage contains metrics for the amount of IO requests.
```

Comment:
Also here. It should be `IO usage` rather than `requests`. There are few more places below.

Reply:
See above.